modules/RateLimiter/src/request_handler.py

```diff
@@ -1,6 +1,8 @@
 import functools
 from redis import Redis, ConnectionPool
-from config import CONNECTION, LIMITING
+from .config import LIMITING, CONNECTION
```

Comment:
difference between this and the previous one?

Reply:
Python requires dot syntax for relative pathing (E.g. `.<filename>`), this just adds that